Composer/packages/client/src/styles.ts

```diff
@@ -53,6 +53,8 @@ export const content = css`
   flex: 1;
   display: flex;
   flex-direction: column;
+
+  label: Content;
```

Comment:
I don't see "label" as a standard CSS property. What's this do?

Reply:
This is a special property for emotion. It just appends this to its generated class name.

This isn't required when using the babel transform (it gets inferred), but since we are using the typescript compiler we need to add label for something human recognizable.

It's not required, but is helpful when debugging instead of looking at class names like `css-sdkj21`.